backend/pyproject.toml

```diff
@@ -44,6 +44,12 @@ lazify = "^0.4.0"
 packaging = "^23.1"
 python-multipart = "^0.0.9"
 pyjwt = "^2.8.0"
+# Fix https://stackoverflow.com/questions/77582651/why-doesnt-numpy-support-pep517-builds
+numpy = [
```

Comment:
interesting, is this added together when you cherry pick?

Reply:
Removed